.github/workflows/test.yaml

```diff
@@ -890,7 +736,7 @@ jobs:
           - supabase
           - supabase-pool
           - planetscale
-        engine: [binary] #['node-api', 'binary']
+        clientEngine: [node-api] #['node-api', 'binary']
```

Comment:
Why the switch / only testing node-api now?

Reply:
https://github.com/prisma/e2e-tests/pull/1954#discussion_r684442790

Reply:
Changed back to `['binary']`